generators/client/templates/angular/src/main/webapp/app/layouts/main/main.component.ts.ejs

```diff
@@ -23,12 +23,15 @@ import { Router, ActivatedRouteSnapshot, NavigationEnd, NavigationError } from '
 import { TranslateService } from '@ngx-translate/core';
 <%_ } _%>
 
+import { AccountService } from 'app/core/auth/account.service';
+
 @Component({
     selector: '<%= jhiPrefixDashed %>-main',
     templateUrl: './main.component.html'
 })
 export class MainComponent implements OnInit {
     constructor(
+        private accountService: AccountService,
```

Comment:
maybe we need to inject it in tests too ?

Reply:
Good idea. I'll try to add Protractor test for that.

Reply:
I was talking about main.component.spec.ts in fact

Reply:
I don't know how to test this in unit tests. I wrote Protractor test, let's see if this is passing CI.

Reply:
Maybe I'm wrong as I'm not a front guy, but let me  try to explain.
I don't think a specific test is needed, but I think we need to inject the `accountService` in tests:
- in `main.component.ts`, we have `private translateService: TranslateService`
- so in `spec.ts` file, we have:
    - https://github.com/jhipster/generator-jhipster/blob/master/generators/client/templates/angular/src/test/javascript/spec/app/layouts/main/main.component.spec.ts.ejs#L39
    - https://github.com/jhipster/generator-jhipster/blob/master/generators/client/templates/angular/src/test/javascript/spec/app/layouts/main/main.component.spec.ts.ejs#L63-L65

That's why I though we need similar thing with this new `accountService`

Reply:
As `AccountService` is part of `TestModule`: https://github.com/jhipster/generator-jhipster/blob/master/generators/client/templates/angular/src/test/javascript/spec/test.module.ts.ejs#L70-L71 and `TestModule` is imported: https://github.com/jhipster/generator-jhipster/blob/master/generators/client/templates/angular/src/test/javascript/spec/app/layouts/main/main.component.spec.ts.ejs#L45 then `MockAccountService` is used, so should be OK.

By the way, CI with added Protractor test succeeded :)

Old code before this change didn't pass that added Protractor test.